locales/de.json

```diff
@@ -310,12 +310,12 @@
     "%A %B %-d, %Y": "%A %B %-d, %Y",
     "(edited)": "(editiert)",
     "YouTube comment permalink": "YouTube-Kommentar Permalink",
-    "permalink": "",
+    "permalink": "permalink",
     "`x` marked it with a ❤": "`x` markierte es mit einem ❤",
     "Audio mode": "Audiomodus",
     "Video mode": "Videomodus",
     "Videos": "Videos",
     "Playlists": "Wiedergabelisten",
-    "Community": "",
+    "Community": "Gemeinschaft",
```

Comment:
I'd change that to `Community`, that's how YouTube itself calls it.

- https://support.google.com/youtube/community?hl=de
- https://support.google.com/youtube/answer/6052538?hl=de